Conversions/OctToBinary.js

```diff
@@ -0,0 +1,18 @@
+function octalToBinary (octal) {
+  let binary = '' // an empty string to store the binary number
+  for (let i = 0; i < octal.length; i++) { // loop through each octal digit
```

Comment:
This could be done more elegantly by `map`ing the octal digits to 3 bits each, then `join`ing the resulting array. This avoids the potentially inefficient (O(n²) assuming it isn't optimized out) repeated string concatenation you're doing here.